mgmt/rest/v2/config.go

```diff
@@ -33,17 +33,51 @@ type PolicyTable cpolicy.RuleTable
 
 type PolicyTableSlice []cpolicy.RuleTable
 
-// cdata.ConfigDataNode implements it's own UnmarshalJSON
 type PluginConfigItem struct {
 	cdata.ConfigDataNode
 }
 
+// PluginConfigResponse represents the response of a plugin config items.
+//
+// swagger:response PluginConfigResponse
+type PluginConfigResponse struct {
```

Comment:
This provides the (generated) client with a response object for plugin config calls.  You will notice that this struct is referenced in api.go (e.g. [here](https://github.com/candysmurf/snap/blob/swagger-1/mgmt/rest/v2/api.go#L141)).